app/basicComponents/SmInput.js

```diff
@@ -15,7 +15,6 @@ const Wrapper = styled.div`
 
 // $FlowStyledIssue
 const Input = styled.input`
-  width: 100%;
```

Comment:
what's the reason to remove it?

Reply:
width 100% is defined in Wrapper and this inherits the correct width. Setting 100% width here overflows the width (can be seen in any of the steps container pages).

Reply:
did you check the latest dev version?

Reply:
pulled and merged with dev. This change was following that.

Reply:
please attach screenshot with overflowed element caused by width: 100%

Reply:
I don't see this issue anymore. With or without width: 100%.